<?xml version="1.0"?>
<launch>

    <param name="total_uav" value="3"/>

    <node pkg="sensor_fusion" type="target.py" name="target" required="True" output="screen">
	</node>
    
    <group ns = "uav1">
        <include file="$(find sensor_fusion)/launch/uav_not_fuse.launch" >
            <arg name="ID" value="1"/>
        </include>
    </group>

    <group ns = "uav2">
        <include file="$(find sensor_fusion)/launch/uav_not_fuse.launch">
            <arg name="ID" value="2"/>
        </include>
    </group>

    <group ns = "uav3">
        <include file="$(find sensor_fusion)/launch/uav_not_fuse.launch">
            <arg name="ID" value="3"/>
        </include>
    </group>



</launch>
    
   